crates/libs/sys/src/Windows/Win32/Graphics/Printing/mod.rs

```diff
@@ -3877,7 +3877,7 @@ pub struct JOB_INFO_2A {
     pub pDriverName: ::windows_sys::core::PSTR,
     pub pDevMode: *mut super::Gdi::DEVMODEA,
     pub pStatus: ::windows_sys::core::PSTR,
-    pub pSecurityDescriptor: *mut super::super::Security::SECURITY_DESCRIPTOR,
+    pub pSecurityDescriptor: *mut ::core::ffi::c_void,
```

Comment:
I wonder what happened here...

Reply:
Agree. Looks like it reverted to `void*` in metadata, investigating.

Reply:
Oh, now I remember why this sounded familiar. I made this change!

This was an incorrect mapping in metadata. In this specific case, `JOB_INFO_2` looks like so:

```cpp
typedef struct _JOB_INFO_2 {
  ...
  LPDEVMODE            pDevMode;
  LPTSTR               pStatus;
  PSECURITY_DESCRIPTOR pSecurityDescriptor;
  ...
```

`PSECURITY_DESCRIPTOR` [is not a pointer to a SECURITY_DESCRIPTOR](https://devblogs.microsoft.com/oldnewthing/20151223-00/?p=92701). It's `typedef PVOID PSECURITY_DESCRIPTOR` behind the scenes.

So this is correct.

Reply:
https://github.com/microsoft/win32metadata/issues/835